cli/displayAdsRecorder.js

```diff
@@ -12,28 +12,28 @@ const findAdsInDirectory = require("../src/util/findAdsInDirectory");
 (async () => {
   program
     .version(packageJson.version)
-    // .option('-g, --glob <data>', 'Globbing pattern like "-p ./src/**/.richmediarc"')
     .option('-c, --chunkSize <data>', 'Define chunkSize', 10)
```

Comment:
Did you happen to take a look at this one? Maybe we can call it jpgChunkSize for now?

Reply:
updated all the functions so now this chunkSize applies to all